lib/distributions/TdCondTdGridDistribution.m

```diff
@@ -0,0 +1,229 @@
+classdef TdCondTdGridDistribution < AbstractGridDistribution
```

Comment:
It would be nice to add a comment to describe what kind of distributon this is exactly.

Reply:
I addressed this. Feel free to merge the latest changes.